src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -958,7 +958,7 @@ internal bool VerifyRemoteCertificate(RemoteCertificateValidationCallback? remot
                 X509Certificate2? certificate = CertificateValidationPal.GetRemoteCertificate(_securityContext, out remoteCertificateStore);
 
                 if (_remoteCertificate != null && certificate != null &&
-                    Enumerable.SequenceEqual(certificate.RawData, _remoteCertificate.RawData))
+                    certificate.RawData.SequenceEqual(_remoteCertificate.RawData))
```

Comment:
I'm actually not sure if this is using the LINQ extension method or the Span version... but I'm guessing the former.

Either way, we should remove the using System.Linq at the top of the file, which will make sure we're using the Span version.

Reply:
agreed. updated.